PerformanceEfficiency/AppServiceAutoscalingSample/PerfStressWebApp/PerfStressWebApp.csproj

```diff
@@ -4,4 +4,17 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(RunConfiguration)' == 'CPUstressWebApp' " />
```

Comment:
I didn't see `CPUstressWebApp` as a configuration in the .sln file.  Is this empty property group needed?

Reply:
Not sure, that line was added by Visual Studio.  I used the builtin package manager to add the 4 nuget packages.

Reply:
I'm going to hold off on merging for some additional testing. It appears my browser was caching some of the files and I need to figure out what to tweak the project settings to actually publish the js and css files that are pulled in via nuget.

Reply:
I'm going to roll back the initial changes and instead just copy over the latest files from the updated project template.